app/src/main/java/mono/hg/fragments/HiddenAppsFragment.java

```diff
@@ -106,7 +106,8 @@ private void loadApps() {
         Collections.sort(availableActivities, new ResolveInfo.DisplayNameComparator(manager));
 
         // Clear the list to make sure that we aren't just adding over an existing list.
-        appList.clear();
+        // hit and try solution for new apps not adding in the list. didn't execpted how but it worked can try.
```

Comment:
This is unrelated to the app list. HiddenAppsFragment _always_ fetch the latest list of apps (thereby not requiring a receiver to listen for any new apps). Removing this line serves very little use.